app/javascript/gobierto_data/webapp/components/Sidebar.vue

```diff
@@ -122,15 +124,29 @@ export default {
           this.slugDataset = this.rawData.data[index].attributes.slug
           this.tableName = this.rawData.data[index].attributes.table_name
 
-          this.$root.$emit('nameDataset', this.titleDataset)
-          this.$root.$emit('sendTableName', this.tableName)
-          this.$root.$emit('sendSlug', this.slugDataset)
-          this.nav(this.slugDataset)
+          this.getMeta()
 
         })
         .catch(error => {
           console.error(error)
-
+        })
+    },
+    getMeta() {
+      this.urlPath = location.origin
+      this.endPoint = `/api/v1/data/datasets/${this.slugDataset}/meta`
+      this.url = `${this.urlPath}${this.endPoint}`
+      axios
+        .get(this.url)
+        .then(response => {
+          this.rawData = response.data
+          this.numberRows = this.rawData.data.attributes.data_summary.number_of_rows
+          this.arrayFormats = this.rawData.data.attributes.formats
+          this.columns = this.rawData.data.attributes.columns
+          this.columns = Object.keys(this.columns)
```

Comment:
When you set `this.` variables, try to do it all in once to avoid rendering. Or `this.columns = Object.keys(this.rawData.data.attributes.columns)` or use an additional variable as 
```
const columns = this.rawData.data.attributes.columns
this.columns = Object.keys(columns)
```